src/3d/shaders/light.inc.frag

```diff
@@ -0,0 +1,222 @@
+const int MAX_LIGHTS = 8;
```

Comment:
Is this from qt3d too? Do we need copyright headers here?

Reply:
Yes this was copied verbatim from qt3d. The original file did not have any copyright header - I will at least put a note about the source and assumed license...